dxp/how_clay_arrives_to_dxp.md

```diff
@@ -0,0 +1,223 @@
+# How Clay arrives to DXP?
+
+The goal of this section is to explain how Clay arrives to DXP, what's the correct way to enable it into your theme or module and what options do we have once imported.
+
+## How is Clay brought into DXP?
+
+There are 2 main methods to import Clay into DXP:
+
+-   as a Theme Inheritance
+-   as a Node Dependency
+
+The Theme Inheritance happens usually when you use a theme generated with [liferay-js-themes-toolkit](https://github.com/liferay/liferay-js-themes-toolkit) and select one existing Liferay theme as a base.
+
+For example, if you are working on a theme based on `style`, `unstyled`, `classic` or `admin` you will inherit Clay and its properties.
+
+The Node Dependency can be usually find in the DXP modules where it is imported directly into the module and don't follows the build process of the themes.
+
+## How is Clay imported into a DXP theme?
+
+In the complex infrastructure of the themes, Clay is imported automatically if you set `style`, `unstyled`, `classic` or `admin`, as a base for your theme, from the generator.
```

Comment:
Clay is automatically imported in `styled`, `classic`, and `admin`. In `unstyled` you have a copy of Clay CSS, but it's not being imported in `clay.scss` or `main.scss`.

Reply:
In the complex infrastructure of the themes -> In the complex infrastructure of themes